components/outlier-detection/isolation-forest/isolation_forest.ipynb

```diff
@@ -273,7 +273,8 @@
    "source": [
     "!helm install ../../../helm-charts/seldon-core --name seldon-core \\\n",
     "        --namespace seldon \\\n",
-    "        --set ambassador.enabled=true"
+    "        --set ambassador.enabled=true \\\n",
+    "        --set ambassador.env.AMBASSADOR_SINGLE_NAMESPACE=true"
```

Comment:
I think this is not strictly necessary as otherwise it will be cluster-wide and that would work too. But previously it was namespace-only.

Reply:
For a little while I wasn't able to see anything in the grafana dashboard but that turned out to be unrelated as I was seeing it with master too and it was addressed by recreating minikube.